lang/lib/charset.py

```diff
@@ -2,10 +2,10 @@
 CUSTOM_CHARS = {
     '\x06': '⏬',
     '\x04': '🔃',
-    '\xe4': 'µ',
+    '\xe4': 'µ', #\xC2\xB5
     '\xdf': '°',
     '\xe1': 'ä',
-    '\xe4': 'μ',
+    #'\xe4': 'μ', #xCE\xBC
```

Comment:
What are we doing here? You cannot have multiple entries in this map. Remove the comment entirely.

Reply:
This is better, but I would still remove completely this line.

Reply:
I only kept is as some reminder. Looks like some tools used by translators use odd/unconventional chars
I will move the comment above as so we keep an eye on it.

Reply:
I see but this is not the place to put it anyway because the map can't contain more than a single entry.

Doesn't U+03bc (the wrong one) fail to build anyway?